.github/CODEOWNERS

```diff
@@ -421,7 +421,7 @@
 # Story project.
 /core/controllers/story*.py @aks681
 /core/domain/story*.py @aks681
-/core/templates/components/entity-creation-services/story-creation.service.ts @kevintab95
+/core/templates/components/entity-creation-services/story-creation.service*.ts @kevintab95
```

Comment:
[Not related to this PR] @kevintab95 Any specific reason for having services in component dir?